Dockerfile

```diff
@@ -90,6 +90,9 @@ RUN install-tool helm v3.12.3
 # renovate: datasource=github-releases lookupName=jsonnet-bundler/jsonnet-bundler
 RUN install-tool jb v0.5.1
 
+# renovate: datasource=github-tags lookupName=NixOS/nix
```

Comment:
https://github.com/containerbase/base/blob/628fa020965b9ed04c8dbf09599df25df6219dfb/test/nix/Dockerfile#L57

Reply:
Thanks. Fixed and amended the commit.

Reply:
Let me know if there are any additional changes required for this PR. Thanks!